### PR TITLE
Revert "will finish second part of 7"

### PR DESCRIPTION
**Pull request by DanteCypress**
Reverts learn-academy-2019-delta/javascript-intermediate-assessment-3-adamcruser#1

### CODE REVIEW

File: js-intermediate.js [hunk 1]
```diff
@@ -1,40 +1,22 @@
-ASSESSMENT 3: INTERMEDIATE JAVASCRIPT
-Coding practical questions
-
-Be sure to use all given test cases
-
-1. Write a function called getFib that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
```
Comment (DanteCypress) [inline js-intermediate.js]: Hey Adam, Take a look at this again. Here's an article, it should have you with the solution.
https://medium.com/quick-code/fibonacci-sequence-javascript-interview-question-iterative-and-recursive-solutions-6a0346d24053

File: js-intermediate.js [hunk 2]
```diff
@@ -63,109 +45,40 @@ var originalArray1 = [1, 2, 3, 4, 5, 6, 7]
...
-// var middleLetters1 = "hello"
-// // Expected output: “l”
-// var middleLetters2 = "lamas"
-// // Expected output: “am”
+// 7. Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
 
 var middleLetters1 = "hello"
-
-let letters=(str)=>{
-let newLetters = str.split('')
```
Comment (DanteCypress) [inline js-intermediate.js]: Give this another shot after we worked it this morning.

**Review by DanteCypress — COMMENTED** [review]
Couple changes

File: js-intermediate.md [hunk 3]
```diff
@@ -5,102 +5,94 @@ Answer the following questions. First, without external resources. Challenge you
...
-  Researched answer: The .class selector selects elements with a specific class attribute. To select elements with a specific class, write a period (.) character, followed by the name of the class. You can also specify that only specific HTML elements should be affected by a class.
-  
-  A class can be used several times, while an ID can only be used once, so you should use classes for items that you know you're going to use a lot.
+  Researched answer:
 
 
 4. Name 4 semantic HTML tags.
 
-  Your answer:not sure what these are.
+  Your answer:
```
Comment (sjproctor) [inline js-intermediate.md:67]: You found examples of semantic tags in the researched section, but to expand on the concept: Older html tags tend to be abbreviated. This became problematic for accessibility devises. The computer can read the tag along with the content for a more descriptive experience. Semantic tags are just tags that are named for how they render content.

File: js-intermediate.md [hunk 4]
```diff
@@ -5,102 +5,94 @@ Answer the following questions. First, without external resources. Challenge you
...
+  Researched answer:
 
 
 3. How would you explain the idea of "inheritance" in object oriented programming?
 
-  Your answer: When working in React or JavaScript you have child/dumb components that will be passed down or will inherit traits from the Parent/Smart component (like a family tree).
+  Your answer:
 
-  Researched answer:Inheritance is a mechanism in which one class acquires the property of another class. For example, a child inherits the traits of his/her parents. With inheritance, we can reuse the fields and methods of the existing class. Hence, inheritance facilitates Reusability and is an important concept of OOPs.
+  Researched answer:
```
Comment (sjproctor) [inline js-intermediate.md:98]: These researched answers are fantastic. I don't have much more to add. You hit the key words and concepts I was looking for. 
- Client-side development
- Variables existing only within the scope of their block
- Reusing methods with inheritance 
Great job!

**Pull request merged**